express/__init__.py

```diff
@@ -78,5 +90,7 @@ def property(self, property_name, *args, **kwargs):
         Returns:
              dict
         """
-        property_instance = self._get_property_class(property_name)(property_name, self.parser, *args, **kwargs)
+        Property_Class = self._get_property_class(property_name)
+        # This is an instance or child of BaseProperty
+        property_instance = Property_Class(property_name, self.parser, *args, **kwargs)
```

Comment:
Not a requirement, but rather than a comment you could just set the type:
```
property_instance: BaseProperty = Property_Class(property_name, self.parser, *args, **kwargs)
```
or else `from typing import Type` and set the type of `Property_Class` instead:
```
Property_Class: Type[BaseProperty] = self._get_property_class(property_name)
```

Reply:
Sprinkled some type hints around __init__